src/components/Card.js

```diff
@@ -10,11 +10,15 @@ const Card = (props) => {
   let q = parse(window.location.search, { ignoreQueryPrefix: true });
   let flat = q.flat === "true" || q.flat === "1";
   let flatStyle = '';
+  let id = null;
   if (flat) {
     flatStyle = 'flat ';
+    if (props.title || props.doc) {
```

Comment:
The navigation at the top of the card page is also implemented as a card.  So we're treating a "real" card as any card that has a title or doc string.

Reply:
Does it make sense to pull this out of the `if (flat)` check?

Reply:
I see, I wasn't sure if these were per page or card.

LGTM